docs/docs/guides/developer_guides/getting_started.md

```diff
@@ -0,0 +1,252 @@
+Secret key:     0x1a9af8f75b3430223b628f7b423667da14304d376a82cfd31dfe138d00a20c61
+Partial address: 0x198022f6ec9b51f3af4cdc731adf606c0143ad35f7947ad604db50e9310a31ac
+Salt:            0x0000000000000000000000000000000000000000000000000000000000000000
+Init hash:       0x04630e3997204b1c417868aa449beb272c3c3bf0efe22315c5920665a6390db7
+Deployer:        0x0000000000000000000000000000000000000000000000000000000000000000
+
+Waiting for account contract deployment...
+Deploy tx hash:  09245ec0d905a461c717d6f70600e3c13b8746ee97785fd491b58f5f56111854
+Deploy tx fee:   200013616
+```
+
+You can double check by running `aztec get-accounts`.
+
+You will be needing that `Address` and `Secret key` fields later! To make it easier to reference, export them like this:
+
+```bash
+export ACCOUNT_ADDRESS=<ADDRESS>
+export SECRET_KEY=<SECRET_KEY>
+```
+Make sure to replace the `<ADDRESS>` and `<SECRET_KEY>` with the actual results that were printed.
+
+With this new account, let's get some local test tokens!
+
+## Deploying a contract
+
+The sandbox comes with some contracts that you can deploy and play with. One of these is an example token contract. 
+
+Deploy it with this:
+
+```bash
+aztec deploy TokenContractArtifact --secret-key $SECRET_KEY --args $ACCOUNT_ADDRESS test-token TST 18
+```
+
+This takes 
+- the contract artifact as the argument, which is `TokenContractArtifact`
+- the secret key of the deployer account, which we exported as `$SECRET_KEY`
+- the args that the contract constructor takes, which is the `admin` (`$ACCOUNT_ADDRESS`), `name` (`test-token`), `symbol` (`TST`), and `decimals` (`18`).
+
+On successful deployment, you should see something like this:
+
+```bash
+ aztec:cli [INFO] Using wallet with address 0x22d82b1ef045dfb4c3e0c4eac41fcf04ba29a3468d5ef5a123e4754064a4038d +0ms
+Contract deployed at 0x1eaa85d27c6ceed91f39a4885e2996f1f6de5d861c40dc9f0cfba43011b15243
+Contract partial address 0x19e957e72ec743a91c341b6e148e41f57c503741ea95e5b6e7158892dcb9f347
+Contract init hash 0x2f4bfd6cfa8f56f133bd0cd5ddfd1cb973436e0d385019491ce8fb950ae7eaf4
+Deployment tx hash: 12e43da89dcb4fbc693419d37e2a01f8f381da49b2cd1ae8492cc91b98aafd08
+Deployment salt: 0x115dd94b02568fc1f875fa782805151557684450f6aee058b28f9f010593e671
+Deployment fee: 200943060
+```
+
+Nice one! Let's export the `Contract deployed at` address so we can use it to call the contract:
+
+```bash
+export CONTRACT_ADDRESS=<CONTRACT_ADDRESS>
+```
+
+Make sure to replace the `<CONTRACT_ADDRESS>` with the actual contract address that was printed.
+
+In the next step, let's mint some tokens!
+
+## Playing with public functions
+
+Call the public mint function like this:
+
+```bash
+aztec send mint_public  --contract-artifact TokenContractArtifact --contract-address $CONTRACT_ADDRESS --secret-key $SECRET_KEY --args $ACCOUNT_ADDRESS 100 
```

Comment:
(an example)

```bash
aztec-wallet send mint_public -ca last --args accounts:new_wallet 10 -f accounts:new_wallet
```

(see the `last` being passed here? the CLI knows what type it expects for `-ca` so it will just resolve the contract aliased as `last` with type `contracts`)